support_branding/demo/ir_config_parameter_data.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <data noupdate="0">
+        <record id="demo_config_parameter_company_name" model="ir.config_parameter">
+            <field name="key">support_company</field>
+            <field name="value">Sunflower IT</field>
+        </record>
+        <record id="demo_config_parameter_company_url" model="ir.config_parameter">
+            <field name="key">support_company_url</field>
+            <field name="value">https://www.sunflowerweb.nl</field>
+        </record>
+        <record id="demo_config_parameter_company_color" model="ir.config_parameter">
+            <field name="key">support_branding_color</field>
+            <field name="value">#fff</field>
+        </record>
+        <record id="demo_config_parameter_support_email" model="ir.config_parameter">
+            <field name="key">support_email</field>
+            <field name="value">info@sunflowerweb.nl</field>
```

Comment:
Let's put "Odoo Community Association", "https://odoo-community.org" and "oca@example.com" here though